index.html

```diff
@@ -18,14 +18,17 @@
 
     <link rel="stylesheet" href="css/common.css">
     <link rel="stylesheet" href="css/first-form.css">
+    
+    <!--IntroJS style -->
+    <link rel="stylesheet" href="node_modules/intro.js/introjs.css" >
   </head>
 
-  <body>
+  <body onload="javascript:introJs().start();">
     <!--[if lt IE 8]>
       <p>You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
 
-    <header class="header-container">
+    <header class="header-container" data-step="1" data-intro="Welcome! Let us go through a short tutorial on how to book a seminar hall">
```

Comment:
You should do the step 1 on the `main-continer`. I have no idea as to why you placed that for the currently "ugly" header ;-)